Magento_Theme/styles/modules/_copyright.scss

```diff
@@ -0,0 +1,7 @@
+.copyright {
+    background-color: $gray-light;
+    color: $white;
+    text-align: center;
+    font-size: $font-size-extra-small;
+    padding: 10px 0;
+}
```

Comment:
we have typography component. maybe we name this component: `bar` ? it's full width block with text inside